api/proto/config/netconfig.proto

```diff
@@ -8,18 +8,20 @@ option java_package = "com.zededa.cloud.uservice.proto";
 
 import "fw.proto";
 import "netcmn.proto";
+import "devcommon.proto";
 
 message NetworkConfig {
-        string id = 1;
-        NetworkType type = 5;
+  string id = 1;
+  UUIDandVersion uuidandversion = 2;
```

Comment:
This results in two UUIDs; one by itself in the id field and one as part of the UUIDandVersion.
That is bad. Better to just add a version field.

Reply:
"id" will be deprecated once this is deployed everywhere.
Till then we can maintain backward compatibility.

Reply:
That would require 1) comment in the proto file e.g., // XXX to be deprecated for the id and 2) code in EVE to use and echo the UUIDandVersion from config to status.

This is woefully incomplete.

Reply:
Agreed on 1).
2) has to be done in 3 steps:
a. change in common protobuf definition
b. EVE controller starts using the new field in backward compatible way (fill up both id and uuidandversion) so that deployed EVE code does not break.
c. EVE makes the change to use new field